contentcuration/contentcuration/frontend/shared/views/Menu.vue

```diff
@@ -0,0 +1,34 @@
+<script>
+
+  import { VMenu } from 'vuetify/lib/components/VMenu';
+
+  export default {
+    name: 'Menu',
+    extends: VMenu,
```

Comment:
Is this `Menu` extending `VMenu` so that you can override prop defaults?

Reply:
Yes, exactly